src/equations/equations.jl

```diff
@@ -374,6 +374,7 @@ include("acoustic_perturbation_2d.jl")
 abstract type AbstractLinearizedEulerEquations{NDIMS, NVARS} <: AbstractEquations{NDIMS, NVARS} end
 include("linearized_euler_2d.jl")
 
-abstract type AbstractEquationsParabolic{NDIMS, NVARS} <: AbstractEquations{NDIMS, NVARS} end
+abstract type AbstractEquationsParabolic{NDIMS, NVARS, GradientVariables} <: AbstractEquations{NDIMS, NVARS} end
+# Here
```

Comment:
What is here?

Reply:
A trailing comment 😅. Will remove it soon.

Reply:
Seems like it's neither here nor there.

Reply:
🙄